vllm/distributed/kv_transfer/kv_connector/v1/nixl_connector.py

```diff
@@ -565,11 +745,60 @@ def add_remote_agent(self, nixl_agent_meta: NixlAgentMetadata):
                      len(blocks_data), engine_id, self.rank)
 
         # Register with NIXL.
-        descs = self.nixl_wrapper.get_xfer_descs(blocks_data, "VRAM")
+        descs = self.nixl_wrapper.get_xfer_descs(blocks_data,
+                                                 self.nixl_memory_type)
         self.dst_xfer_side_handles[
             engine_id] = self.nixl_wrapper.prep_xfer_dlist(
                 self._remote_agents[engine_id], descs)
 
+    def sync_recved_kv_to_device(self, req_id: str):
+        """copy recved kv from host buffer to device."""
+        if not self.use_host_buffer:
+            return
+        assert self.h2d_copy_blocks is not None
+
+        if req_id in self._recving_metadata and \
+           req_id not in self._recving_transfers:
+            meta = self._recving_metadata[req_id]
+            # local decode only
+            if not meta.do_remote_prefill:
+                return
```

Comment:
should here be continue or return?

Reply:
Since only one request is handled by this function, "continue" or "return" will be the same.